pkg/microservice/aslan/core/common/repository/models/image_tags.go

```diff
@@ -0,0 +1,21 @@
+package models
+
+import "go.mongodb.org/mongo-driver/bson/primitive"
+
+type ImageTags struct {
+	ID          primitive.ObjectID `bson:"_id,omitempty" json:"id,omitempty"`
+	RegistryID  string             `bson:"registry_id" json:"registry_id"`
+	RegProvider string             `bson:"reg_provider" json:"reg_provider"`
+	ImageName   string             `bson:"image_name" json:"image_name"`
+	Namespace   string             `bson:"namespace" json:"namespace"`
+	ImageTags   []*ImageTag        `bson:"image_tags" json:"image_tags"`
+}
+
+type ImageTag struct {
+	TagName string `bson:"tag_name" json:"tag_name"`
```

Comment:
i suggest record digest in this struct since digest is the real factor which determines the creation time